crates/common/fork_choice/src/store.rs

```diff
@@ -775,8 +775,11 @@ pub fn get_forkchoice_store(
     anchor_state: BeaconState,
     anchor_block: BeaconBlock,
     db: ReamDB,
+    anchor_state_root: Option<B256>,
```

Comment:
I don't think passing the state root here is justified for something that only happens once initially. This seems like a premature optimization